web/views.py

```
@@ -3468,13 +3527,35 @@
     if request.method == "POST":
         form = ChallengeSubmissionForm(request.POST)
+        challenge_detail = {
+            "title": challenge.title,
+            "description": challenge.description,
+            "student_answer": request.POST.get("submission_text"),
+        }
+
         if form.is_valid():
+            try:
+                ai_response = ai_assignment_corrector(challenge_detail)
+            except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)